cishouseholds/validate_class.py

```diff
@@ -52,6 +83,14 @@ def produce_error_column(self):
         self.error_column_list = []
 
     def filter(self, return_failed: bool, any: bool, selected_errors: List = []):
+        """
+        Returns Dataframe with validation column filtered by the errors wanted. TODO: doublecheck
+        Parameters
+        ----------
```

Comment:
return_failed: return a subset of the failed rows
all: do all conditions need to be met to be considered an erroneous row or just any?
selected_errors: list of errors that denote a failed row